lcls-twincat-optics/lcls_twincat_optics_plc/POUs/Main.TcPOU

```diff
@@ -0,0 +1,109 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.12">
+  <POU Name="Main" Id="{3ee7fb27-9ab5-458e-ba31-7ac39aacb9ee}" SpecialFunc="None">
+    <Declaration><![CDATA[PROGRAM Main
+VAR
+	// Axes for each system
+	// M1L0
+	M1 : DUT_MotionStage:=(nCommand:=3); // Yup
```

Comment:
Can we use `ENUM_EpicsMotorCmd.EPICS_MOTOR_MOVE_ABS` instead of `3`?

Reply:
I'd like to note that you shouldn't set this argument at all in a deployed project, but it's perfectly fine as a testing aid.

Reply:
Resolved in commit 69e2191.

Just some clarification, does setting `nCommand:=3` (or the equivalent ENUM in lcls-twincat-motion) allow for control via the NC gui, but in production this shouldn't be set at all because `lcls-twincat-motion` is setting it as part of the epics control?

Reply:
Oh and I meant to add, is this also the case for `DUT_MotionStage.nEnableMode`, i.e for production this should not be explicitly set but is adjusted through some epics enable motor field?

Reply:
`nCommand` should be changed only by the IOC, or by the engineer if emulating the IOC's functions. It doesn't affect your ability to move via NC.

`nEnableMode` will allow you move via NC if set to "Always". It should be set to whatever you need for the application, or "Always" for testing.

Reply:
A quick addendum: you can also put `nEnableMode` to `Never` for testing if you'd like to enable/disable from the NC interface instead of from the PLC code.